packages/workbox-precaching/src/PrecacheController.ts

```diff
@@ -35,83 +33,55 @@ declare global {
   }
 }
 
+interface PrecacheControllerOptions {
+  cacheName?: string;
+  plugins?: WorkboxPlugin[];
+  fallbackToNetwork?: boolean;
+}
+
 /**
  * Performs efficient precaching of assets.
  *
  * @memberof module:workbox-precaching
  */
 class PrecacheController {
-  private readonly _cacheName: string;
-  private readonly _urlsToCacheKeys: Map<string, string>;
-  private readonly _urlsToCacheModes: Map<string, "reload" | "default" | "no-store" | "no-cache" | "force-cache" | "only-if-cached">;
-  private readonly _cacheKeysToIntegrities: Map<string, string>;
-
-  private _router?: Router;
-  private _strategy?: Strategy;
-  private _installed?: boolean = false;
-  private readonly _cacheKeyPlugin: WorkboxPlugin;
-  private readonly _plugins: WorkboxPlugin[] = [];
+  private _installAndActiveListenersAdded?: boolean = false;
```

Comment:
Since it's initialized to `false`, I don't think the `?` annotation is appropriate.

Reply:
I kept the `?` and didn't initialize to false to save a few bytes.